README.md

```diff
@@ -49,37 +65,33 @@ npm run build
+### 2. Install the Quantic Library
+
+[Install the Coveo Quantic Library as a Salesforce unlocked package](https://docs.coveo.com/en/quantic/latest/usage/#install-quantic).
+
+### 3a. Deploy the Project Using the Org Development Model
 
 The Org Development Model allows you to connect directly to a non-source-tracked org (sandbox, Developer Edition (DE) org, Trailhead Playground, or even a production org) to retrieve and deploy code directly. This model is similar to the type of development you have done in the past using tools such as Force.com IDE or MavensMate.
 
 To start developing with this model in Visual Studio Code, see [Org Development Model with VS Code](https://forcedotcom.github.io/salesforcedx-vscode/articles/user-guide/org-development-model). For details about the model, see the [Org Development Model](https://trailhead.salesforce.com/content/learn/modules/org-development-model) Trailhead module.
 
 When working with non-source-tracked orgs, use the commands `SFDX: Deploy Source to Org` (VS Code) or `sfdx force:source:deploy` (Salesforce CLI) and `SFDX: Retrieve Source from Org` (VS Code) or `sfdx force:source:retrieve` (Salesforce CLI). The `Push` and `Pull` commands work only on orgs with source tracking (scratch orgs).
 
-### 2b. Pushing the code in a Scratch Org
+### 3b. Push the Code in a Scratch Org
 
 When working with source-tracked orgs, use the commands `SFDX: Push Source to Org` (VS Code) or `sfdx force:source:push` (Salesforce CLI) and `SFDX: Pull Source from Org` (VS Code) or `sfdx force:source:pull` (Salesforce CLI). Do not use the `Retrieve` and `Deploy` commands with scratch orgs.
 
-### 2c. Installing the app using an Unlocked Package
+### 3c. Install the App Using an Unlocked Package
 
 TBD
 
-### 3. How to enable the Case Flow in your community
+### 4. Enable the Case Flow in Your Community
 
-1. In the class CaseAssistEndpoint. Change this line to add your case assist ID:
-   `public static final String CASE_ASSIST_ID = 'YOUR_CASE_ASSIST_ID';`
-1. In your Salesforce community, drag the Lightning Flow component onto a Community page and select the Case_Assist_Demo_Flow shipped with this repository.
-1. In the published version of your community, fill in the subject and description fields **(minimum of 10 characters in the description)** in the first screen to see the Related Category section appear and get predictions for the Case Reason field.
+1. In your Salesforce community, drag the Lightning Flow component in a Community page and select the `Case_Assist_Recommended_Flow` or the `Case_Assist_Demo_Flow` shipped with this repository.
+2. After selecting the name of the flow, two inputs labeled `caseAssistId` and `engineId` appear. Fill the `caseAssistId` input with your [Case Assist Id](https://docs.coveo.com/en/3328/#retrieving-a-case-assist-id) retrieved from your Case Assist Configuration and fill the `engineId` input with an appropriate name for the engine that the Quantic components will register to.
```

Comment:
Is there any doc we can refer to regarding the Quantic engineID?

Reply:
Didn’t find any 🧐

Reply:
This [Usage](https://docs.coveo.com/en/quantic/latest/usage/#javascript) section mentions it but doesn't describe it.
This reference doc for the [QuanticCaseAssistInterface](https://docs.coveo.com/en/quantic/latest/reference/case-assist-components/case-assist-case-assist-interface/) has a little description for the argument.

Reply:
Thanks! We can use that👌